pocs/observatory.py

```diff
@@ -23,7 +23,7 @@
 
 class Observatory(PanBase):
 
-    def __init__(self, cameras=None, scheduler=None, dome=None, *args, **kwargs):
+    def __init__(self, cameras=None, scheduler=None, dome=None, mount=None, *args, **kwargs):
```

Comment:
Updates to the docstring are needed. Can either be here or separate PR to get all of them.